_posts/2017-10-19-the-road-to-devops-nirvana.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "The Road To DevOps Nirvana"
+date:   2017-10-19 02:03:58 +0000
+categories: devops enterprise transformation
+author: Hamish Tedeschi
+image: img/devops/the devops transformation helix.png
+---
+
+IT Transformation is not a phrase to be thrown around lightly. The mere mention of it will have some people shuddering at the thought of multi million dollar programmes attempting to shoehorn new organisational structures and / or technology into an organisation. And in many cases failing to achieve what it sets out to do, which was to fundamentally change the way an organisation operates.
```

Comment:
Feels like the article jumps between high level IT Transformation concepts and low level detail.  Who's the post aimed at?

Reply:
I have no idea. I think I was high when I wrote it.